libs/barista-components/formatters/src/duration/duration-formatter.ts

```diff
@@ -31,12 +31,14 @@ import {
  * @param formatMethod the formatting method
  * @param outputUnit dtTimeUnit | undefined value describing the unit to which it should format e.g to seconds
  * @param inputUnit dtTimeUnit value describing which unit the duration is in (default: milliseconds)
+ * @param maxDecimals max amount of decimals
  */
 export function formatDuration(
   duration: number,
   formatMethod: DurationMode = 'DEFAULT',
   outputUnit?: DtTimeUnit,
   inputUnit: DtTimeUnit = DtTimeUnit.MILLISECOND,
+  maxDecimals?: number,
```

Comment:
I think we should add this parameter also to the pipe DtDuration. Since this function should be identical to the pipe. We need to add a default to the pipe though

Reply:
Could you also add this to the documentation README then.

Reply:
right now we don't have a default, that's why I left it unset, so we would not add any change to the current users. But if you want I can set it to 3 decimals, for instance.

Reply:
I totally agree that it should not change anything, so maybe having it as undefined is fine as a default and just not do anything in the case it is not provided. My point was mainly about having the option on the pipe to define it as well. I am sorry i was not clear about that :)